test/e2e_node/device_manager_test.go

```diff
@@ -266,6 +272,131 @@ var _ = SIGDescribe("Device Manager  [Serial] [Feature:DeviceManager][NodeFeatur
 		})
 
 	})
+
+	ginkgo.Context("With sample device plugin", func(ctx context.Context) {
```

Comment:
You should not pass context as an argument to the anonymous function.
From the `node-e2e-containerd` CI lane failure:
```
W1227 11:58:05.859]   Invalid Function
W1227 11:58:05.859]   test/e2e_node/device_manager_test.go:276
W1227 11:58:05.859]     [Container] node must be passed func() - i.e. functions that take nothing and
W1227 11:58:05.860]     return nothing.  You passed func(context.Context) instead.
W1227 11:58:05.860] 
W1227 11:58:05.860]     Learn more at: http://onsi.github.io/ginkgo/#node-decorators-overview
```